### PR TITLE
Coverage for test_static_path_deprecated and test_static_url_path

### PR DESCRIPTION
**Pull request by jphilipsen05**
Made tests in tests_basic.py

### CODE REVIEW

File: tests/test_basic.py [hunk 1]
```diff
@@ -1116,6 +1116,25 @@ def test_static_files():
     rv.close()
 
 
+def test_static_path_depricated():
```
Comment (jeffwidman) [inline tests/test_basic.py]: Typo should be `deprecated` (without an "i")

Comment (jphilipsen05) [inline tests/test_basic.py]: That's what I thought and how I typed it until I used it from this doc:
http://pytest.org/latest/recwarn.html#ensuring-a-function-triggers-a-deprecation-warning

and the python builtin spells it with an e. So I'm fine either way. Do you need me to push it again or is this something that you were able to fix/change if you need?

Comment (ThiefMaster) [inline tests/test_basic.py]: Simply amend the commit and force-push

Comment (jphilipsen05) [inline tests/test_basic.py]: Thanks for fixing that. Were you able to fix it or do I need to fix and push again?

**Comment by ThiefMaster** [comment]
Check your git config to ensure your email is correct - it's either wrong or just not linked to your GitHub account. The latter is not a big issue but it's nicer if your commits are attributed to your GitHub user!

**Comment by jphilipsen05** [comment]
Typo is fixed.

**Comment by ThiefMaster** [comment]
It'd be better to amend the commit - no need to have separate commits for this

**Pull request merged**